recipes/full_finetune_distributed.py

```diff
@@ -833,7 +833,8 @@ def train(self) -> None:
                                     else self._optim_ckpt_wrapper
                                 ),
                             ),
-                            "tokens_per_second_per_gpu": num_tokens / time_per_step,
+                            "tokens_per_second_per_gpu": num_tokens
+                            / (time_per_step * world_size),
```

Comment:
why times world size? This is per gpu, so we should not multiply * numbers of gpu